CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [BUGFIX] Ingester: Prevent any reads on TSDBs once `Ingester.closeAllTSDB()` has been called. #3350
```

Comment:
Few things please:
- Could you rebase `master` and, after rebasing, move the change log entry under "master / unreleased"?
- Please specify the PR number #4304 instead of the issue number #3350
- We're used to sort CHANGELOG entries by type: CHANGE, FEATURE, ENHANCEMENT, BUGFIX

Reply:
Thanks for letting me know. Should be updated now.